clients/fides-js/src/components/Overlay.tsx

```diff
@@ -210,19 +216,20 @@ const Overlay: FunctionComponent<Props> = ({
           })
         : null}
       {options.fidesEmbed ? (
-        <ConsentContent
-          titleProps={attributes.title}
-          className="fides-embed"
-          i18n={i18n}
-          renderModalFooter={() =>
-            renderModalFooter({
-              onClose: handleCloseModalAfterSave,
-              isMobile: false,
-            })
-          }
-        >
-          {renderModalContent()}
-        </ConsentContent>
+        bannerIsOpen ? null : (
```

Comment:
👍 nice, so this prevents the banner and modal from showing at the same time within the container